shared/chat/conversation/input.desktop.js

```diff
@@ -43,6 +43,7 @@ class Conversation extends Component<void, Props, State> {
       const {selectionStart = 0, selectionEnd = 0} = this._input.selections() || {}
       const nextInputText = [inputText.substring(0, selectionStart), emojiColons, inputText.substring(selectionEnd)].join('')
       this.props.setInputText(nextInputText)
+      this._input.focus()
```

Comment:
@chrisnojima it's just cut out of the diff, but this line is inside an `if (this._input) {` block.